python/hail/methods/tests.py

```diff
@@ -100,3 +100,12 @@ def test_mendel_errors(self):
         self.assertEqual(ind.key, ['s'])
         self.assertEqual(var.key, ['v'])
         dataset.annotate_rows(mendel=var[dataset.v]).count_rows()
+
+    def test_concordance(self):
+        dataset = self.get_dataset()
+        glob_conc, cols_conc, rows_conc = methods.concordance(dataset, dataset)
+        print(glob_conc[1][4])
+        print(glob_conc[4][0])
+        print(glob_conc[:][3])
+        cols_conc.write('/tmp/foo.kt', overwrite=True)
+        rows_conc.write('/tmp/foo.kt', overwrite=True)
```

Comment:
I assume this wasn't supposed to stay in?

Reply:
Why? Table has a `write` method.

Reply:
Sorry, that window is misleading, I meant the entire test. There are print statements, and no assertions.